### DISCUSSION TITLE
how to display emoj in md file?

### DISCUSSION THREAD

**Discussion by CaptainJon** [remarkjs/remark#973] [Q&A] [answered]
I really can't understand the introduction about how to use the emoj plugin, right now I am using typora and the emoj is like ": smile :", so how to make it work after it been shown on website. right now it just shows as string.

**Comment by wooorm** [comment, marked as answer]
https://github.com/remarkjs/remark-gemoji

**Reply by CaptainJon** [reply]
i am using the lib 'react-markdown' to display markdown files, i dunno how to work with this. i used aixos to get md content, but the emoj couldnt be parsed. then i tried the way that descrited in your linked page, but it shows an error "fs__WEBPACK_IMPORTED_MODULE_0___default.a.readFile is not a function", my code is like follow:

async function main() {
    const file = await unified()
      .use(remarkParse)
      .use(remarkGemoji)
      .use(remarkStringify)
      .process(await read("../../assets/markdowns/platform-rules.md"))

    console.log(String(file))
  }

the error shows just in this way. nothing in the log, just error

**Reply by ChristianMurphy** [reply]
If you want the emoji to display using `react-markdown`, apply the remark plugin to `react-markdown` using the `remarkPlugins` prop https://github.com/remarkjs/react-markdown#api
